### PR TITLE
Qdm 56 update

### PR DESCRIPTION
**Pull request by adongare**
- Updated cqm-execution to support QDM5.6
- Have not published models. Planning to refer direct git commit hashes in package json. Once we have one regression test done in bonnie, we'll publish models and execution. Let me know if that is okay  

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: MAT-2648 Update cqm-execution to QDM 5.6
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cqm-execution",
-  "version": "3.1.0",
+  "version": "4.0.0",
```
Comment (serhii-ilin) [inline package.json:3]: does it overlap with cqm-execution/fhir?

Comment (adongare) [reply package.json:3]: not now but it will some point. It is 4.0.1 there. We have not released it yet. I think We should make FHIR one 5.x or 6.x

Comment (dczulada) [reply package.json:3]: I think the FHIR one being 5.x would work.  It could be interesting if there is a new QDM version post 5.x, but that could still be published as 6.x if need be.

**Review by serhii-ilin — APPROVED** [review]

**Review by dczulada — APPROVED** [review]
